### PR TITLE
Update use case in DG and format in UG

### PR DESCRIPTION
**Pull request by wxy1203**
### Changes: ###
- mark task use cases in DG
- mark task user guide

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -450,36 +450,23 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
-* 3a. The given persons index is invalid.
+* 1a. The given task index is invalid.
 
-    * 3a1. Arrow shows an error message.
+    * 1a1. Arrow shows an error message.
 
       Use case resumes at step 2.
       
-* 4a. The tasks list is empty.
+* 2a. The tasks list is empty.
```
Comment (yilinzyl) [inline docs/DeveloperGuide.md:442]: I think we could either choose to remove this extension
or add new steps in our MSS to 1. User requests to list tasks 2. Arrow shows a list of tasks

**Review by yilinzyl — APPROVED** [review]
Small error that has been resolved in my previous pull request. LGTM!

**Review by kianseong — APPROVED** [review]
LGTM!

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (wxy1203) [reply docs/DeveloperGuide.md:442]: Removed.

**Pull request merged**